docs/README.md

```diff
@@ -1,33 +1,93 @@
-# Website
+<p align="center">
+  <a href="https://coderoad.github.io/">
+    <img alt="CodeRoad Logo" src="https://user-images.githubusercontent.com/9423525/89562564-26b4f780-d7e8-11ea-9eb0-3109b0e55d3b.png" width="100" />
+  </a>
+</p>
+<h1 align="center">
+CodeRoad VSCode
+</h1>
 
-This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+<p align="center">
+  <a href="#" alt="Version"><img src="https://vsmarketplacebadge.apphb.com/version/Coderoad.coderoad.svg" /></a>
+  <a href="#" alt="Installs"><img src="https://vsmarketplacebadge.apphb.com/installs/Coderoad.coderoad.svg" /></a>
+   <a href="#" alt="Downloads"><img src="https://vsmarketplacebadge.apphb.com/downloads/Coderoad.coderoad.svg" /></a>
+</p>
 
-### Installation
+CodeRoad is a VSCode extension that allows you to play interactive coding tutorials in your editor.
 
-```
-$ yarn
-```
+![CodeRoad Image](./docs/static/img/tutorial-example.png)
```

Comment:
Path is correct, image just not being displayed in preview. Should work when merged